package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klarna/ui",
-  "version": "1.17.2",
+  "version": "1.17.3-rc.0",
```

Comment:
Oh we don't actually do this. Versions should be updated in master with `npm version` so that tags are added and so on.